api_utils/botEdit.py

```
@@ -0,0 +1,296 @@
+    "تعريب": ["لا للتعريب"],
+    "تقييم آلي": ["لا للتقييم الآلي"],
+    "reftitle": ["لا لعنونة مرجع غير معنون"],
+    "fixref": ["لا لصيانة المراجع"],
+    "cat": ["لا للتصنيف المعادل"],
+    "stub": ["لا لتخصيص البذرة"],
+    "tempcat": ["لا لإضافة صناديق تصفح معادلة"],
+    "portal": ["لا لربط البوابات المعادل", "لا لصيانة البوابات"],
+}
+
+
+def bot_May_Edit_do(text="", title_page="", botjob="all"):
```

Pylint (3.3.7)

[convention] Function name "bot_May_Edit_do" doesn't conform to snake_case naming style

(C0103)